src/shared/const.ts

```diff
@@ -14,6 +14,9 @@ export enum ExternalUrl {
 export const LEDGER = {
   // Ledger now uses bip44 derivation path
   // For more details, please check - https://ledger.readthedocs.io/en/latest/background/hd_use_cases.html
-  GET_BTC_MAINNET_ADDRESS: "173'/0'/0'/0/0",
-  GET_BTC_TESTNET_ADDRESS: "173'/1'/0'/0/0"
+  BTC_DERIVE_PATH: {
+    MAINNET: "84'/0'/0'/0/0",
+    TESTNET: "84'/1'/0'/0/0"
+  },
```

Comment:
Make [derive_path](https://github.com/xchainjs/xchainjs-lib/blob/38645063168663fb23b364d48ae7698efa1ab37c/packages/xchain-bitcoin/src/client.ts#L58-L61) of xchain-btc public to import it here. See https://github.com/xchainjs/xchainjs-lib/issues/134


@sarawut11 Does it make sense to do it similar with `BNB_DERIVE_PATH_ARRAY` (extracting into `xchain-binance` etc.) ?